### PR TITLE
fix HTTPS Anywhere conflict

### PR DESCRIPTION
**Pull request by bbodenmiller**
Fix URL redirect conflict discussed at https://github.com/Jdhaimson/smilealways/issues/27.

Fix https://github.com/Jdhaimson/smilealways/issues/27

### CODE REVIEW

**Comment by bbodenmiller** [comment]
@Jdhaimson any progress getting this merged? Been running for the last 9 days without any problems.

**Comment by bbodenmiller** [comment]
@Jdhaimson any chance you can review, merge, and publish? Been working great.

File: chromeplugin/request.js [hunk 1]
```diff
@@ -31,22 +30,15 @@ function detectRedirect(details) {
                + "|(read.amazon.com)"
                + "|(login.amazon.com)"
                + "|(payments.amazon.com)"
-               + "|(amazon.com/clouddrive)";
+               + "|(amazon.com/clouddrive)"
+               + "|(http://)"; //all Amazon pages now redirect to HTTPS, also fixes conflict with HTTPS Everywhere extension
```
Comment (Timothee) [inline chromeplugin/request.js]: Should this start with `^` maybe? I don't know if Amazon has that but I'm thinking that there could be cases where another URL is passed as a query parameter or something like that.

Maybe not a big deal.

Comment (bbodenmiller) [inline chromeplugin/request.js]: Hmm. I haven't see anywhere where another URL is passed as a query parameter.

**Comment by bbodenmiller** [comment]
Thanks! Any idea when you might get it published to Chrome Store?

**Comment by ialexryan** [comment]
Really looking forward to the new version being pushed to the Chrome store, @delitzer and @Jdhaimson! That little orange exclamation point has been bugging me for more than 6 months now 😔

**Comment by bbodenmiller** [comment]
@delitzer @Jdhaimson any chance to upload to Chrome store? Do you need someone else to take over?

**Comment by schwabe** [comment]
@bbodenmiller I uploaded a forked version that includes amazon.de and also this fix (https://chrome.google.com/webstore/detail/smile-always-internationa/)

**Comment by delitzer** [comment]
Hey folks, sorry for the delay here and thanks so much for getting involved. I've taken over maintenance from Josh and just pushed an updated version to the Chrome store. It should be live within an hour.

**Comment by ialexryan** [comment]
Thank you, @delitzer! Much appreciated.

**Pull request merged**